Index: lib/main.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/main.dart b/lib/main.dart
new file mode 100644
--- /dev/null	(date 1744371978459)
+++ b/lib/main.dart	(date 1744371978459)
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Ulin Mahoni App',
+      home: Scaffold(
+        appBar: AppBar(
+          title: Text('Home Page'),
+        ),
+        body: Center(
+          child: Text('Welcome to Ulin Mahoni!'),
+        ),
+      ),
+    );
+  }
+}
